test/integration.test.js

```diff
@@ -44,6 +44,9 @@ for (const integrationTest of fs.readdirSync(__dirname + "/integration")) {
   // ignore a few tests known to fail on windows
   if (process.platform === 'win32' && skipOnWindows.includes(integrationTest)) continue;
 
+  // ignore leveldown.js test on macOS, https://github.com/Level/leveldown/issues/801
+  if (process.platform === 'darwin' && integrationTest === 'leveldown.js') continue;
```

Comment:
It is the only test failing for macOS, might make sense to disable it? Might be related to https://github.com/Level/leveldown/issues/801.

Reply:
Sounds good to me 👍

Reply:
(I've refactored the skip, to be the same way as for the tests skipped on Windows)